scripts/DocGenerator.php

```diff
@@ -352,6 +422,14 @@ private function buildReturns($returns)
         return $returnsArray;
     }
 
+    public function buildMagicReturns($return)
+    {
+        return [
+            'types' => $this->handleTypes([$return]),
+            'description' => null
```

Comment:
This method is not needed anymore. Descriptions will work for magic method returns!